packages/component/src/Utils/detectBrowser.js

```diff
@@ -0,0 +1,10 @@
+const { userAgent } = navigator;
+
+// not DRY so that list can be alphabetical
```

Comment:
Huh?

Reply:
Ignore, you can't read regex. xD

Reply:
I was referring to the comment. 'not DRY'?